evanxd.md

```diff
@@ -1,8 +1,19 @@
-## 07/25 ~ 07/29 ##
+## 08/01 ~ 08/05 ##
 
 * [Firefox]
   - Dev Tools
-    - Bug 1286892 - Inspector sidebar tabs now have a bigger font-size, r+ and landed
-    - Bug 1282791 - Reps: uninteresting props algorithm is wrong, landed
+    - Bug 1289912 - Can't scroll in JSON Viewer in Nightly, landed
```

Comment:
Could you help me by document the status on a new line? This will make diff more readable.